wolfSSL/mkdocs-ja.yml

```diff
@@ -0,0 +1,164 @@
+site_name: wolfSSL Manual
+site_url: https://wolfssl.com/
+docs_dir: build/html/
+site_dir: html/ja
```

Comment:
To be consistent with the wolfSSH and wolfBoot manuals, should we just have the HTML output directory be the html directory?

```
site_dir: html/
```

Reply:
Moved it to html. Let me know if it bothers English version.